### PR TITLE
Add missing docstrings in variables.jl

### PR DESCRIPTION
**Pull request by odow**
First step in #1790. Before removing exports, we should check that all the ones we want to export are documented.

### CODE REVIEW

File: src/variables.jl [hunk 1]
```diff
@@ -367,10 +371,11 @@ end
 """
     set_lower_bound(v::VariableRef,lower::Number)
```
Comment (mlubin) [inline src/variables.jl]: Nit: missing space after comma

File: src/variables.jl [hunk 2]
```diff
@@ -350,6 +349,11 @@ end
 
 # lower bounds
 
+"""
+    has_lower_bound(v::VariableRef)
+
+Return `true` if `v` has a lower bound.
```
Comment (mlubin) [inline src/variables.jl]: Cross-reference to `LowerBoundRef`?

File: src/variables.jl [hunk 3]
```diff
@@ -421,6 +426,11 @@ end
 
 # upper bounds
 
+"""
+    has_upper_bound(v::VariableRef)
+
+Return `true` if `v` has a upper bound.
```
Comment (mlubin) [inline src/variables.jl]: Cross-ref to `UpperBoundRef`?

File: src/variables.jl [hunk 4]
```diff
@@ -492,6 +503,11 @@ end
 
 # fixed value
 
+"""
+    is_fixed(v::VariableRef)
+
+Return `true` if `v` is a fixed variable.
```
Comment (mlubin) [inline src/variables.jl]: Cross-ref to `FixRef`?

File: src/variables.jl [hunk 5]
```diff
@@ -651,6 +679,11 @@ function set_binary(variable_ref::VariableRef)
     return
 end
 
+"""
+    unset_binary(variable_ref::VariableRef)
+
+Remove the binay constraint on the variable `variable_ref`.
```
Comment (mlubin) [inline src/variables.jl]: typo: binary

File: src/variables.jl [hunk 6]
```diff
@@ -615,13 +639,17 @@ function unset_integer(variable_ref::VariableRef)
...
-                                                     integer_index(v),
-                                                     ScalarShape())
+    return ConstraintRef{Model, MOIINT, ScalarShape}(
+        owner_model(v), integer_index(v), ScalarShape())
 end
 
+"""
+    is_binary(v::VariableRef)
+
+Return `true` if `v` is a binary variable.
```
Comment (mlubin) [inline src/variables.jl]: Better to say "is constrained to be binary" than "binary variable".

File: src/variables.jl [hunk 7]
```diff
@@ -607,21 +634,36 @@ function set_integer(variable_ref::VariableRef)
     return
 end
 
+"""
+    unset_integer(variable_ref::VariableRef)
+
+Remove an integrality constraint on the variable `variable_ref`.
```
Comment (blegat) [inline src/variables.jl]: an -> the
There can be only one integrality constraint for the variable

Comment (blegat) [reply src/variables.jl]: The is used for `unset_binary`

File: docs/src/variables.md [hunk 8]
```diff
@@ -592,4 +606,29 @@ in the model. This is useful for performing operations like:
 owner_model
 VariableRef
 all_variables
+
```
Comment (mlubin) [inline docs/src/variables.md]: Do the empty lines affect display at all?

**Pull request merged**